docs/SimpleFormIterator.md

```diff
@@ -94,6 +95,10 @@ This prop lets you pass a custom element to replace the default Add button.
 </SimpleFormIterator>
 ```
 
+## `clearButton`
+
+This prop lets you pass a custom element to replace the default "Clear the list" button
```

Comment:
That's not true anymore

Reply:
Removed and `disableClear` description is added : b987e628a3c25ea157aaa022f81781f4055abf5c